gp_emulator/GaussianProcess.py

```diff
@@ -3,6 +3,8 @@
 import numpy as np
 import scipy.spatial.distance as dist
 import random
+import sys
+#import _gpu_predict
```

Comment:
Never use commenting-out to control code configuration. You can make python import statements conditional under if statements, to test if the library is available and import if so.